content/en/docs/setup/install/istioctl/index.md

```diff
@@ -278,62 +278,76 @@ For example, the following command overrides the `pilot.traceSampling` Helm conf
 $ istioctl manifest apply --set values.pilot.traceSampling=0.1
 {{< /text >}}
 
-Helm values can also be set in an `IstioControlPlane` definition as described in
+Helm values can also be set in an `IstioOperator` definition as described in
 [Customize Istio settings using the Helm API](#customize-istio-settings-using-the-helm-api), below.
 {{< /tip >}}
 
-### Identify an Istio feature or component
+### Identify an Istio component
 
-The `IstioControlPlane` API groups control plane components by feature, as shown in the table below:
+The `IstioOperator` API defines components as shown in the table below:
 
-| Feature | Components |
-|---------|------------|
-`base` | `CRDs`
-`trafficManagement` | `pilot`
-`policy` | `policy`
-`telemetry` | `telemetry`
-`security` | `citadel`, `nodeAgent`, `certManager`
-`configManagement` | `galley`
-`gateways` | `ingressGateway`, `egressGateway`
-`autoInjection` | `injector`
-`coreDNS` | `coreDNS`
-`thirdParty` | `cni`
+| Components |
+| ------------|
+`base` |
+`pilot` |
+`proxy` |
+`sidecarInjector` |
+`telemetry` |
+`policy` |
+`citadel` |
+`nodeagent` |
+`galley` |
+`ingressGateways` |
+`egressGateways` |
+`cni` |
 
-In addition to the core Istio components, third-party addon features and components are also available. These can only
-be enabled and configured through the Helm pass-through API:
+In addition to the core Istio components, third-party addon components are also available. These can 
+be enabled and configured through the addonComponents spec of `IstioOperator` API or Helm pass-through API:
 
-| Feature | Components |
-|---------|------------|
-`telemetry` | `prometheus`, `prometheusOperator`, `grafana`, `kiali`, `tracing`
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  addonComponents:
+    grafana:
+      enabled: true
+{{< /text >}}
 
-Features can be enabled or disabled, which enables or disables all of the components that are a part of the feature.
-Namespaces that components are installed into can be set by component, feature, or globally.
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    grafana:
+      enabled: true
+{{< /text >}}
 
-### Configure the feature or component settings
+### Configure the component settings
```

Comment:
Changing this title changes the id of the generated html section, so you need to fix the broken link.

Reply:
```
 - ./public/docs/setup/upgrade/istioctl-upgrade/index.html
  *  linking to /docs/setup/install/istioctl/#configure-the-feature-or-component-settings, but configure-the-feature-or-component-settings does not exist (line 65535)
     <a href="/docs/setup/install/istioctl/#configure-the-feature-or-component-settings">IstioControlPlane API Custom Resource Definition</a>
```

Reply:
done